tests/api/api-resources.test.js

```diff
@@ -0,0 +1,131 @@
+// Copyright Contributors to the Open Cluster Management project
+
+jest.retryTimes(global.retry)
+
+const { fail } = require('assert');
+const { performance } = require('perf_hooks');
+
+const squad = require('../../config').get('squadName')
+const {
+  getKubeConfig,
+  getSearchApiRoute,
+  getToken,
+  searchQueryBuilder,
+  sendRequest,
+} = require('../common-lib/clusterAccess')
+
+const {
+  fetchAPIResourcesWithListWatchMethods,
+  formatResourcesFromSearch,
+  formatFilters,
+  getResourcesFromOC,
+  getClusterList,
+  shouldUseAPIGroup
+} = require('../common-lib/index')
+
+const { sleep } = require('../common-lib/sleep')
+
+// Set list to ignore resources that aren't being collected by Search.
+const ignoreKindResourceList = ['clusterclaim', 'event']
+
+// Set list of resources that require filtering by api group.
+const requireAPIGroup = []
+
+/**
+ * Base test for kubernetes
+ * @param {*} kind
+ * @param {*} apigroup
+ * @param {*} cluster
+ * @param {*} user
+ * @param {*} namespace
+ */
+function baseTest(kind, apigroup, cluster = { type: 'hub', name: 'local-cluster' }, user = 'kubeadmin', namespace = '--all-namespaces') {
+  var runTest = test
+
+  try {
+    var expectedResources = getResourcesFromOC(kind, apigroup, namespace, cluster)
+  } catch (err) {
+    runTest = test.skip
```

Comment:
Let's log that we are skipping the test and the reason why.
```suggestion
    console.error(`Skipping test because there was an error getting the expected resources from OC command. kind=${kind} apigroup=${apigroup} namespace=${namespace} cluster=${cluster}  Error: {err}`)
    runTest = test.skip
```

Reply:
Added console error to the api test.